tiledb/cloud/client.py

```diff
@@ -5,8 +5,8 @@
 from . import config
 from . import tiledb_cloud_error
 from .rest_api import ApiException as GenApiException
-import urllib
```

Comment:
I think this was unintentionally dropped here, which is causing the CI failure.

Reply:
Whoops! Fixed.